### PR TITLE
BoxStation: Added preloaded drone shell dispenser to Testing Lab

### PR DESCRIPTION
**Pull request by coiax**
Drone shells will now be present roundstart, regardless if R&D are
doing their job or not.

There are drones already on the ~~derelicht~~ derelict, which can easily
make their way to the mainstation anyway.

:cl: coiax
rscadd: Boxstation, not to be outdone by all the other competing station designs, has installed a drone shell dispenser in the Testing Lab, which makes automated drones to help fix the station.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
:-1: we've had this discussion before and it has ultimately failed not to mention this is bad placement for it if you were going to do it

**Comment by coiax** [comment]
I'm reopening the discussion, dammit.

And where would you put it?

**Comment by Bawhoppen** [comment]
misc testing lab right above the firealarm remove the locker

**Comment by Core0verload** [comment]
>I don't like it. Ghost roles mustn't be guaranteed.

**Comment by Bawhoppen** [comment]
@Core0verload drones aren't on the same par as other ghost roles however, they're a minor side thing where only a handful of people enjoy that kind of experience
besides, ghost roles will be basically guaranteed after space ruins stuff is finished

**Comment by coiax** [comment]
There are guaranteed ghost roles on the Derelict right now. At any point in a round, if one's left, you can take a shell, go to mainstation, do R&D, print drone shells. Probably take about ten minutes tops. But that's just jumping through a lot of pointless hoops to make drone shells for the station. The only difference between a drone shell dispenser and doing the current solution is that the current solution takes less metal and glass and has like a ten minute setup time.

**Comment by jughu** [comment]
DRONES WASTING MUH METAL

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -45408,7 +45408,6 @@
 	dir = 1;
 	pixel_y = -24
 	},
-/obj/structure/closet,
 /obj/item/pipe{
```
Comment (ChangelingRain) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: Should probably remove this too.

Comment (coiax) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: Remove what?

Comment (ChangelingRain) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: The pipe item.

**Comment by Cheridan** [comment]
Roundstart drones sounds fine.

Roundstart golems are just bad because it's removing randomization from a feature based heavily around it for the purpose of maintaining novelty.

**Comment by coiax** [comment]
It's technically two seperate PRs, Cheridan.

It wasn't actually planned to do these together, I've always wanted to put a drone shell dispenser on the map, and Free Golems happened during the map freeze.

**Comment by coiax** [comment]
Are there any objections to this aside from objecting to guaranteed  ghost roles?

Three drone shells are located on the derelict in constant places, which is constant regardless of main station map. As I said before, the time to set up and print shells on the main station is about 10 minutes.

Here's a thought, what if we put the dispenser there, but set the initial cooldown to say, ten minutes? Then the drones don't appear immediately, but at a predictable time into the round.

**Comment by TrustyGun** [comment]
the rest of em merged already pls merge

**Comment by RandomMarine** [comment]
This should be testmerged for a while at least. See how things go with roundstart drones. If things go well someone can map out a better spot for the dispenser.

**Comment by coiax** [comment]
@RandomMarine: The current spot was suggested by @Bawhoppen, have you got a better one in mind?

**Comment by RandomMarine** [comment]
There's tons of space in maintenance that would work, for instance, set up a room in that space gap in engineering maintenance.

**Comment by Bawhoppen** [comment]
@RandomMarine no, everyone's thought immediately drifts to being in maint, but thats just a plain bad idea, its the best if its actually in a real place where people can refill it

**Comment by TrustyGun** [comment]
i think it may work best here:
untitled

**Comment by Bawhoppen** [comment]
I will fight to the death for it not to be in maint. Putting it in maint is autistic

**Comment by TrustyGun** [comment]
Why not place it in maint?

**Comment by coiax** [comment]
> the rest of em were merged already pls merge

**Comment by GunHog** [comment]
Why do you care so much where it is? Misc. Testing or Maint, it does not seem to matter much, just merge it as is, assuming everything is tested and working.

**Comment by coiax** [comment]
Bawhoppen literally picked the location where it is, he has no objections.

**Comment by GunHog** [comment]
I have no problem with it being there. Merge it!

**Pull request merged**